clients/application-amqp/src/test/java/org/eclipse/hono/application/client/amqp/ProtonBasedRequestResponseCommandClientTest.java

```diff
@@ -133,6 +134,50 @@ public void testSendCommandSucceeds(final VertxTestContext ctx) {
         AmqpClientUnitTestHelper.assertReceiverLinkCreated(connection).handle(protonDelivery, response);
     }
 
+    /**
+     * Verifies that sending a command results in a failed Future if the command response contains an error status.
+     *
+     * @param ctx The vert.x test context.
+     */
+    @Test
+    public void testSendCommandFailsForErrorStatusResponse(final VertxTestContext ctx) {
+        final String subject = "setVolume";
+        final String replyId = UUID.randomUUID().toString();
+        final Map<String, Object> properties = Map.of("appKey", "appValue");
+        final int commandStatus = HttpURLConnection.HTTP_BAD_REQUEST;
+        final String responseContentType = MessageHelper.CONTENT_TYPE_APPLICATION_JSON;
```

Comment:
`responseContentType` is not used anywhere and it would be good to remove it.

Reply:
I've removed it.